File: Python/2309.py
import sys

f = lambda : sys.stdin.readline().rstrip()

dwarf = list()
height = 0

for i in range(9):

    temp = int(f())
    dwarf.append(temp)
    height += temp;

dwarf = sorted(dwarf)
tf = False
for i in range(9):

    for j in range(i+1, 9):

        value = dwarf[i] + dwarf[j]
        if (height - value) == 100:

            dwarf[i] = 0
            dwarf[j] = 0
            tf = True
            break

    if tf: break

for i in dwarf:

    if i == 0: continue
    print(i)

File: Java/_2309.java
package Algorithm.BaekJoon;

import java.io.*;
import java.util.Arrays;

public class _2309 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] dwarf = new int[9];
        int sum = 0;

        for(int i = 0; i < 9; i++){

            int temp = Integer.valueOf(br.readLine().trim());
            dwarf[i] = temp;
            sum += temp;
        }

        Arrays.sort(dwarf);

        boolean tf = false;
        for(int i = 0; i < 9; i++){

            for(int j = i+1; j < 9; j++){

                int value = dwarf[i] + dwarf[j];
                if(sum - value == 100){

                    dwarf[i] = 0;
                    dwarf[j] = 0;
                    tf = true;
                    break;
                }
            }

            if(tf) break;
        }

        for(int x : dwarf){

            if(x == 0) continue;
            System.out.println(x);
        }
    }
}
